Detectors/TPC/spacecharge/include/TPCSpaceCharge/DataContainer3D.h

```diff
@@ -144,6 +144,7 @@ bool DataContainer3D<DataT, Nx, Ny, Nz>::initFromFile(TFile& inpf, const char* n
     return false;
   }
   mData = dataCont->mData;
+  delete dataCont;
```

Comment:
I think the same should be done a few lines further down?

Reply:
You mean the loadFromFile() function? There the pointer is just returned and I think shouldnt be deleted? But since this function is actually never used in any of the SpaceCharge classes, it might be better to just remove the function and keep the class more simple.